### ISSUE TITLE
Remove [markdown] from the python scripts

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#651]
We currently have "code like this"

```
# %% [markdown]
# ### Code completion and help in Jupyter notebook 
# * **Code completion** type `napari_viewer.` and press `TAB`
# * **Help** type `napari_viewer.add_image` and press `SHIFT-TAB` this will open a help associated to the add_image method/command
```

I find this very hard to digest, even as a teacher. The point is that we mainly **have** to look at it like this, because the main idea is to _copy and paste_ text from the activity into a notebook:

<img width="1271" alt="image" src="https://github.com/NEUBIAS/training-resources/assets/2157566/dffc3477-dac5-4865-9641-33954d72a80f">

The `[markdown]` only makes sense if anyone would open the whole file in a jupyter notebook that has jupytext installed, which I think is not the main use-case of this training material, is it?!

I would vote for simplifying this, i think we can keep the `# %%` because that nicely serves to structure the lesson but otherwise I would be for removing any visual clutter from the code.

Any opinions? @tibuch @k-dominik @maulakhan @jhennies

**Comment by k-dominik** [comment]
I think there are at least three ways of learning/teaching with this material

1.) Full on self-study - There I'd expect learners to open the activities in jupyter. However, currently I'd say they are not easy to find at all so that learners maybe might be more inclined in copy pasting - and then, indeed the markdown extra comments could lead to irritation. The text of the comments is helpful, so I'd say it should remain in any case.
2.) Demo-teaching - Could be done by opening the notebooks from the python files - Instructors I would assume would be capable of finding/copying the files before a course. There the markdown cells make for a nicer notebook
3.) Carpentry-style type along teaching - Most if not everything is typed live with the learners - the trainer would be the only person intended to see the python files there.

So for the user groups:
* for learners, it might be a bit irritating to see these extra comments
* for trainers, I think one should be able to deal/ignore with those.

_To conclude_, for me it's more or less a non-issue.

I have bigger concerns with varying coding styles in the material, to be honest.

**Comment by k-dominik** [comment]
I can live with this, of course. For the removal I think it makes sense then to at least make sure the _previous_ markdown cells are included in some of the code cells as comments...

As for the code style I typically go for black (as most of the community these days) - but I understand this is a topic worth of a discussion and should be decided by the community. So I'd say we don't change it for the course (I'll _naturally_ will type in black style though ;)).

**Comment by tischi** [comment]
> For the removal I think it makes sense then to at least make sure the previous markdown cells are included in some of the code cells as comments...

Of course! They contain critical information for the teaching. I just want to make them easier to read.

**Comment by tischi** [comment]
> As for the code style I typically go for black 

Could you please create an issue for this with a link to the black style?

**Comment by tischi** [comment]
Old assessments:

```
  ### 2-D image inspection
    Open image
    [xy_8bit__nuclei_noisy_different_intensity.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xy_8bit__nuclei_noisy_different_intensity.tif).
    Hint: For certain exercises the inspection of the histogram will help
    1. What is the value of the pixel at the indices (x=20,y=20)?
    1. What is the highest value in the image?
    1. What is the most frequently occurring value in the image?
    1. Where is this pixel with the indices (x=0,y=0)? Why is this potentially confusing?
    1. How many pixels does this image have in the x direction?
    1. What is the highest pixel index in the x direction?
    1. If you were to rotate the image by 90 degrees, would it change the image histogram?

    > ## Solution
    > 1. 82
    > 1. 129
    > 1. 55
    > 1. Top left; normally x/y coordinate systems have their origin at the bottom left
    > 1. 59
    > 1. 58
    > 1. No, the gray value histogram is independent of the pixel locations
  {: .solution}

  ### 3-D image inspection
    Open image: [xyz_8bit__mri_head.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xyz_8bit__mri_head.tif)
    1. What is the value of the voxel at the indices (x=93,y=124,z=13)?
    1. Which is the highest value in the image?

    > ## Solution
    > 1. 47
    > 1. 255
    {: .solution}
```

**Comment by jhennies** [comment]
> As for the code style I typically go for black (as most of the community these days) - but I understand this is a topic worth of a discussion and should be decided by the community. So I'd say we don't change it for the course (I'll _naturally_ will type in black style though ;)).

I checked out the black coding style and I'm all for it :)

**Comment by manerotoni** [comment]
Overall what @tischi call a script is in fact a notebook and as such it has been designed. 
The reason for having  #%%  was to paste the data to a file and open in a notebook in a click when using the jupytext extension.
The cells are rendered correctly. 

This avoids the annoying part of copying bit of the code cell by cell. 

I think one could remove the markdown stuff and just leave comments, with the cell decorators. Markdown could be nice if you want to include an image or something more fancy to explain the concepts, also links look nicer.

**Comment by manerotoni** [comment]
There are cases that are so simple that do not require a notebook. In this case one can read it as a single script.

**Comment by tischi** [comment]
@manerotoni for the recent course we changed it now such that we leave the `# %%` but remove all other formatting. We think this is a good compromise. Would you agree?

**Comment by manerotoni** [comment]
Yes I this is a good compromise. You can use it as a script or load it in a notebook.
